instrumentation/rack/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-rack
 
+### v0.21.1 / 2022-10-04
+
+* FIXED: Bring http.request.header and http.response.header in line with … 
```

Comment:
```suggestion
* FIXED: Bring http.request.header and http.response.header in line with semconv 
* ADDED: Add Net::HTTP :untraced_hosts option
```

Reply:
```suggestion
* FIXED: Bring http.request.header and http.response.header in line with semconv
```

"Add Net::HTTP :untraced option" [happened to net_http](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/134/files#r987251443), not rack. 😉